packages/tools/devtools/devtools-core/src/DecomposedContainer.ts

```diff
@@ -61,22 +63,18 @@ export interface DecomposedIContainer extends IEventProvider<IContainerEvents> {
 }
 
 /**
- * Implementation of {@link DecomposedIContainer} that wraps an {@link IFluidDataStoreRuntime}.
- * This class provides a bridge between {@link IFluidDataStoreRuntime} and the devtools system by exposing runtime properties and events as {@link IContainer} interfaces.
+ * Implementation of {@link DecomposedContainer} that wraps an {@link @fluidframework/datastore-definitions/internal#IFluidDataStoreRuntime}.
+ * This class provides a bridge between {@link @fluidframework/datastore-definitions/internal#IFluidDataStoreRuntime} and the devtools system by exposing runtime properties and events.
  *
- * @alpha
+ * @internal
  */
-export class DecomposedContainer
+class DecomposedContainerForDataStore
```

Comment:
We shouldn't need to package export this, so it shouldn't need the `@internal` release tag.